e2store/src/utils.rs

```diff
@@ -47,3 +47,46 @@ pub fn underlying_io_error_kind(error: &Error) -> Option<io::ErrorKind> {
     }
     None
 }
+
+const ERA_DIR_URL: &str = "https://mainnet.era.nimbus.team/";
+
+/// Fetches era file download links
+pub async fn get_era_file_download_links(http_client: &Client) -> anyhow::Result<Vec<String>> {
```

Comment:
> nit: Are we sure there is no api that we can use that returns better structured content (e.g. json), instead of parsing HTML ourself?

Not that I am aware of

> nit: Maybe return HashMap<u64, String>, where key is the epoch number (keep the logic of processing this file in one place)?

sure